boards/forms.py

```diff
@@ -10,4 +10,14 @@ def save(self, *args, **kwargs):
 		board_name = data["board_name"]
 		user = args[0]
 		new_board = Board(name=board_name,owner=user)
-		new_board.save()
\ No newline at end of file
+		new_board.save()
+
+	def update(self, *args, **kwargs):
+		data = self.cleaned_data
+		board_name = data["board_name"]
+		board_id = args[0]
+		update_board = Board.objects.get(pk=board_id)
```

Comment:
instead of passing board_id as kwargs, just pass the board instance instance, so don't need to get the object back again.